app/controllers/application_controller.rb

```diff
@@ -28,6 +28,37 @@ def after_sign_out_path_for(resource_or_scope)
     end
   end
 
+  protected
+
+  # volunteer/supervisor/casa_admin controller uses to send acct creation SMS
+  # returns appropriate flash notice for SMS
+  def send_sms(phone_number, resource_name)
```

Comment:
- This should be generalized to work with all SMS's sent from a controller. aka take in different message bodies.
- Would avoid the naming overlap with TwilioService's 'send_sms' method. Maybe this could be something like  `deliver_sms_to(phone_number, body)`